src/main/java/jimmy/mcgymmy/model/food/Food.java

```diff
@@ -79,20 +86,20 @@ public boolean equals(Object other) {
         Food otherFood = (Food) other;
 
         return otherFood != null
-                && this.getName().equals(otherFood.getName())
-                && this.getProtein().equals(otherFood.getProtein())
-                && this.getCarbs().equals(otherFood.getCarbs())
-                && this.getFat().equals(otherFood.getFat());
+            && this.getName().equals(otherFood.getName())
+            && this.getProtein().equals(otherFood.getProtein())
+            && this.getCarbs().equals(otherFood.getCarbs())
+            && this.getFat().equals(otherFood.getFat());
```

Comment:
I think the standard for 2103 uses 8 spaces instead iirc. Same for Food.toString method